monai/transforms/croppad/dictionary.py

```diff
@@ -434,7 +439,8 @@ def __init__(
     def randomize(self, img_size: Sequence[int]) -> None:
         self._size = fall_back_tuple(self.roi_size, img_size)
         if self.random_size:
-            self._size = [self.R.randint(low=self._size[i], high=img_size[i] + 1) for i in range(len(img_size))]
+            max_size = img_size if self.max_roi_size is None else fall_back_tuple(self.max_roi_size, img_size)
+            self._size = [self.R.randint(low=self._size[i], high=max_size[i] + 1) for i in range(len(img_size))]
```

Comment:
Errors will happen if `low > high`, thus I suggest add a comparison between `roi_size` and `max_roi_size` in `__init__`, and could you please add include one more test case to cover this situation.

Reply:
added check for min / max ROI size.